packages/devtools_app/lib/src/screens/deep_link_validation/deep_links_controller.dart

```diff
@@ -498,9 +498,9 @@ class DeepLinksController extends DisposableController
   }
 
   Future<List<LinkData>> _validateDomain(
-    List<LinkData> linkdatas,
+    List<LinkData> rawLinkdatas,
```

Comment:
super-nit: this Dart code should fit on one line if we remove the trailing comma